packages/components/atoms/Button/Button.modules.scss

```diff
@@ -1,45 +1,90 @@
+@import '../../../stylesheets/base/variables';
+@import '../../../stylesheets/base/colors';
+
+@mixin button-styles {
+  position: relative;
+  display: inline-block;
+  padding: baseUnit(1.25) baseUnit(2);
+  border-radius: baseUnit(0.75);
+  cursor: pointer;
+  text-align: center;
+  text-decoration: none;
+  transition: background-color 0.3s ease, color 0.3s ease;
+  border: none;
+}
+
+@mixin button-hover {
+  &::after {
```

Comment:
Question. Why did you use after here? Is it necessary?

Reply:
Like, I get what you're doing with the opacity, but wouldn't it be simpler to just modify the color of the main button element?

Reply:
Cool